### DISCUSSION TITLE
Questions on a working setup.

### DISCUSSION THREAD

**Discussion by FabPell980** [strongswan/strongswan#1497] [General]
Hello,

I need some explanations on my setup.
Scenario
client -> firewall#1 -> Internet -> Firewall #2 - Ubuntu 22.04.1 Strongswan

This lab setup is working but I have questions:
my config

```
conn mycon
  #firewall #2
  #left=192.168.42.235
  leftid=@swan
  #The network behind this server
  leftsubnet=10.0.0.0/24
  #firewall #1
  right=192.168.42.251
  rightid=@fw
  #The network behind remote firewall #1
  rightsubnet=192.168.168.0/24
  #Connection parameters
  keyexchange=ikev1
  authby=psk
  ike=aes256-sha256-modp2048
  esp=aes256-sha256
  ikelifetime=8h
  lifetime=8h
  auto=start
```
so, if I enable left=192.168.42.235 Swan will not send any packets to firewall 2, not receiving any packets.
on IkeV2 I have the error "received unsupported IKE version 15.4 from 192.168.42.251" 
  
Why this is happening?
  
Thank you

**Comment by tobiasbrunner** [comment]
> so, if I enable left=192.168.42.235 Swan will not send any packets to firewall 2, not receiving any packets.

Then that IP address is presumably not correct (the local address only has to be specified in special scenarios, so not setting `left` is perfectly fine).

> on IkeV2 I have the error "received unsupported IKE version 15.4 from 192.168.42.251"

The other peer probably doesn't support IKEv2 then (what it returns is weird, though, might not even be an IKE message).
